<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QStackedWidget" name="clPage">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>600</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>맑은 고딕</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="currentIndex">
    <number>4</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>80</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>맑은 고딕</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Main</string>
     </property>
    </widget>
    <widget class="QPushButton" name="goQnA">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>180</y>
       <width>160</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>맑은 고딕</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>QnA</string>
     </property>
    </widget>
    <widget class="QPushButton" name="goQuiz">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>270</y>
       <width>160</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>맑은 고딕</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Quiz</string>
     </property>
    </widget>
    <widget class="QPushButton" name="goConsulting">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>360</y>
       <width>160</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>맑은 고딕</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>상담</string>
     </property>
    </widget>
    <widget class="QPushButton" name="goStudy">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>90</y>
       <width>160</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>맑은 고딕</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>개인 학습</string>
     </property>
    </widget>
    <widget class="QPushButton" name="logOut">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>530</y>
       <width>160</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>맑은 고딕</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>로그아웃</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>120</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>개인 학습</string>
     </property>
    </widget>
    <widget class="QPushButton" name="goMain">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>530</y>
       <width>160</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>메인 홈</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_3">
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>120</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>QnA</string>
     </property>
    </widget>
    <widget class="QPushButton" name="goMain_2">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>30</y>
       <width>160</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>메인 홈</string>
     </property>
    </widget>
    <widget class="QTabWidget" name="tabWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>761</width>
       <height>511</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>질문하기</string>
      </attribute>
      <widget class="QTextEdit" name="question_content">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>70</y>
         <width>431</width>
         <height>341</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>70</y>
         <width>71</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>질문</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QPushButton" name="writingComplete">
       <property name="geometry">
        <rect>
         <x>530</x>
         <y>420</y>
         <width>81</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>작성완료</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>20</y>
         <width>71</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>제목</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLineEdit" name="question_title">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>20</y>
         <width>431</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>질문내역</string>
      </attribute>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>370</x>
         <y>30</y>
         <width>51</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>제목</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>380</x>
         <y>60</y>
         <width>51</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>질문</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>380</x>
         <y>260</y>
         <width>51</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>답변</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QTableWidget" name="question_table">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>20</y>
         <width>321</width>
         <height>421</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="autoScrollMargin">
        <number>16</number>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::SingleSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <column>
        <property name="text">
         <string>번호</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>학생</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>질문</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>답변여부</string>
        </property>
       </column>
      </widget>
      <widget class="QTextBrowser" name="show_answer">
       <property name="geometry">
        <rect>
         <x>380</x>
         <y>290</y>
         <width>341</width>
         <height>121</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
      </widget>
      <widget class="QTextBrowser" name="show_question_content">
       <property name="geometry">
        <rect>
         <x>380</x>
         <y>90</y>
         <width>341</width>
         <height>121</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
      </widget>
      <widget class="QTextBrowser" name="show_question_title">
       <property name="geometry">
        <rect>
         <x>420</x>
         <y>30</y>
         <width>301</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
      </widget>
      <widget class="QTextBrowser" name="textBrowser">
       <property name="geometry">
        <rect>
         <x>525</x>
         <y>220</y>
         <width>191</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>440</x>
         <y>220</y>
         <width>81</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>질문한 날짜</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QTextBrowser" name="textBrowser_3">
       <property name="geometry">
        <rect>
         <x>525</x>
         <y>420</y>
         <width>191</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
      </widget>
      <widget class="QLabel" name="label_20">
       <property name="geometry">
        <rect>
         <x>440</x>
         <y>420</y>
         <width>81</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>답변한 날짜</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="page_4">
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>120</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Quiz</string>
     </property>
    </widget>
    <widget class="QPushButton" name="goMain_3">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>530</y>
       <width>160</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>메인 홈</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_5">
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>120</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>상담</string>
     </property>
    </widget>
    <widget class="QPushButton" name="goMain_4">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>30</y>
       <width>160</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>메인 홈</string>
     </property>
    </widget>
    <widget class="QListWidget" name="chat_list">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>140</y>
       <width>511</width>
       <height>321</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLineEdit" name="send_chat">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>480</y>
       <width>351</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_send_chat">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>480</y>
       <width>61</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>전송</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_6">
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>120</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>로그인</string>
     </property>
    </widget>
    <widget class="QPushButton" name="logIn">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>380</y>
       <width>160</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>로그인</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>220</y>
       <width>120</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>이름</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="studentName">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>220</y>
       <width>200</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="fail_message">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>350</y>
       <width>161</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>studentName</tabstop>
  <tabstop>logIn</tabstop>
  <tabstop>goStudy</tabstop>
  <tabstop>goQnA</tabstop>
  <tabstop>goQuiz</tabstop>
  <tabstop>goConsulting</tabstop>
  <tabstop>logOut</tabstop>
  <tabstop>goMain</tabstop>
  <tabstop>question_title</tabstop>
  <tabstop>question_content</tabstop>
  <tabstop>writingComplete</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>goMain_2</tabstop>
  <tabstop>goMain_3</tabstop>
  <tabstop>goMain_4</tabstop>
  <tabstop>question_table</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
